server/integration_tests/nlnext_test.py

```diff
@@ -74,7 +73,6 @@ def run_sequence(self,
                    check_debug_info=True):
     ctx = {}
     for i, q in enumerate(queries):
-      time.sleep(5)
```

Comment:
Wonder why remove this? (I don't know why it exists, perhaps Bo might know)

Reply:
This was a change I had made on my local end but forgot to remove. I had previously discussed this with Bo. The reason I need this is that my local integration tests keep timing out while I am connecting from a non-US VM. But since this only impacts me, there is no reason it should slow down everyone else :)